src/CMakeLists.txt

```diff
@@ -317,14 +317,14 @@ list(APPEND LIBS ${LIBXML2_LIBRARIES})
 add_definitions(${LIBXML2_DEFINITIONS})
 
 # used to launch a tiny http server, which could be used by OAuth services
-find_package(LibSoup2)
-if(LibSoup2_FOUND)
-  include_directories(SYSTEM ${LibSoup2_INCLUDE_DIRS})
-  list(APPEND LIBS ${LibSoup2_LIBRARIES})
-  add_definitions(${LibSoup2_DEFINITIONS})
+pkg_check_modules(LIBSOUP3 libsoup-3.0)
```

Comment:
If there is no FindLibSoup3.cmake shipped w/ CMake (or libsoup3 doesn't ship CMake config files), we should create one. Don't like the idea of using pkg_* macros directly in our main CMake recipe...